jetbrains-core/it/software/aws/toolkits/jetbrains/services/lambda/java/JavaTestUtils.kt

```diff
@@ -69,6 +69,9 @@ private fun copyGradleFiles(fixtureRule: HeavyJavaCodeInsightTestFixtureRule) {
 private fun copyPath(fixtureRule: HeavyJavaCodeInsightTestFixtureRule, root: Path, path: Path) {
     if (path.isDirectory()) {
         Files.list(path).forEach {
+            if (it.fileName.toString().startsWith(".")) {
```

Comment:
Do you think we'll ever have to handle dotfiles?

Reply:
It's just for unit tests, I opened the gradle/ folder and Mac made a .DS_Store which failed the tests 😢